sdgym/benchmark.py

```diff
@@ -1576,7 +1577,9 @@ def _store_job_args_in_s3(output_destination, job_args_list, s3_client):
     job_args_key = f'{path}{job_args_key}' if path else job_args_key
 
     serialized_data = cloudpickle.dumps(job_args_list)
-    s3_client.put_object(Bucket=bucket_name, Key=job_args_key, Body=serialized_data)
+    compressed = gzip.compress(serialized_data, compresslevel=1)
```

Comment:
The `s3_client.put_object` is not designed for large objects, and the `job_arg_list` file is 500MB+ for multi-table dataset, so we compress it here.

Reply:
this is good for now 👍 I think in the future we should push towards not uploading all of the arguments and instead passing them as strings to the cloud instance. The cloud instance can then download or create anything it needs to